room_assistance/indico_room_assistance/util.py

```diff
@@ -7,9 +7,27 @@
 
 from __future__ import unicode_literals
 
+from indico.modules.events.requests.models.requests import Request, RequestState
+
+from indico_room_assistance.definition import RoomAssistanceRequest
+
 
 def is_room_assistance_support(user):
     from indico_room_assistance.plugin import RoomAssistancePlugin
     if user.is_admin:
         return True
     return RoomAssistancePlugin.settings.acls.contains_user('room_assistance_support', user)
+
+
+def event_has_room_with_support_attached(event):
+    from indico_room_assistance.plugin import RoomAssistancePlugin
+    return event.room in RoomAssistancePlugin.settings.get('rooms_with_assistance')
+
+
+def can_request_assistance_for_event(event):
+    has_room_assistance_request = (event.requests
+                                   .filter(Request.type == RoomAssistanceRequest.name,
```

Comment:
nothing wrong with this, but `filter_by(type=.., state=...)` would be more concise here.